scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1546,6 +1550,33 @@ then
     parse_sst_info "$sst_file_info_path" sst galera-gtid "" > "$XB_GTID_INFO_FILE_PATH"
 
     DONOR_BINLOGNAME=$(parse_sst_info "$sst_file_info_path" sst binlog-name "")
+    DONOR_MYSQL_VERSION=$(parse_sst_info "$sst_file_info_path" sst mysql-version "")
+
+    if [[ -n "$DONOR_MYSQL_VERSION" ]]; then
+        local_version_str=""
+        donor_version_str=""
+
+        # Truncate the version numbers (we want the major.minor version like "5.6", not "5.6.35-...")
+        local_version_str=$(expr match "$MYSQL_VERSION" '\([0-9]\+\.[0-9]\+\)')
+        donor_version_str=$(expr match "$DONOR_MYSQL_VERSION" '\([0-9]\+\.[0-9]\+\)')
+
+        # Is this node's pxc version < donor's pxc version?
+        if ! check_for_version $local_version_str $donor_version_str; then
+            wsrep_log_error "******************* FATAL ERROR ********************** "
+            wsrep_log_error "FATAL: PXC is receiving an SST from a node with a higher version."
+            wsrep_log_error "This node's PXC version is $local_version_str.  The donor's PXC version is $donor_version_str."
+            wsrep_log_error "Upgrade this node before joining the cluster."
+            wsrep_log_error "****************************************************** "
+            exit 2
+        fi
+
+        # Is the donor's pxc version < this node's pxc version?
+        if ! check_for_version $donor_version_str $local_version_str; then
+            wsrep_log_warning "WARNING: PXC is receiving an SST from a node with a lower version."
+            wsrep_log_warning "This node's PXC version is $local_version_str. The donor's PXC version is $donor_version_str."
+            wsrep_log_warning "Run mysql_upgrade to upgrade the data."
```

Comment:
Run mysql_upgrade in non-cluster (standalone mode) to upgrade. Check the upgrade process here https://www.percona.com/doc/percona-xtradb-cluster/LATEST/howtos/upgrade_guide.html